core/modules/field/modules/text/text.module

```diff
@@ -220,6 +228,23 @@ function text_field_formatter_settings_form($field, $instance, $view_mode, $form
       '#required' => TRUE,
       '#description' => t('If the summary is not set, the trimmed %label field will be shorter than this character limit.', array('%label' => $instance['label'])),
     );
+    $element['word_boundary'] = array(
+      '#title' => t('Trim on word boundary'),
+      '#type' => 'checkbox',
+      '#default_value' => $settings['word_boundary'],
+      '#description' => t('Trim to the nearest word, rather than the nearest paragraph, line break, or sentence.'),
+    );
+    $element['ellipsis'] = array(
+      '#title' => t('Add ellipsis'),
+      '#type' => 'checkbox',
+      '#default_value' => $settings['ellipsis'],
+      '#description' => t('Add an ellipsis (...) to the end of the trimmed text.'),
```

Comment:
The actual ellipsis is `…` - not `...`. But this text is short enough to be the title/label of the checkbox, instead of being a help/description.